.config/dictionaries/technical.dic

```diff
@@ -307,6 +314,7 @@ tsdx
 tungstenite
 typegen
 typeorm
+uatom
```

Comment:
What is an uatom?

Reply:
it's part of benchmarks. Not sure if it's intended or a typo @seunlanlege ?
```rs
let denom = "transfer/channel-15/uatom";
```